runner/runners/invoke.go

```diff
@@ -133,17 +136,16 @@ func (inv *Invoker) run(cmd *runner.Command, id runner.RunID, abortCh chan struc
 
 	switch st.State {
 	case execer.COMPLETE:
-		srcToDest := map[string]string{
-			stdout.AsFile(): "STDOUT",
-			stderr.AsFile(): "STDERR",
+		tmp, err := temp.NewTempDir("", "invoke")
+		if err != nil {
+			return runner.ErrorStatus(id, fmt.Errorf("error staging ingestion: %v", err))
 		}
-		// TODO(jschiller): get consensus on design and either implement or delete.
-		// if cmd.SnapshotPlan != nil {
-		// 	for src, dest := range cmd.SnapshotPlan {
-		// 		srcToDest[checkout.Path()+"/"+src] = dest // manually concat to preserve src *exactly* as provided.
-		// 	}
-		// }
-		snapshotID, err := inv.filer.IngestMap(srcToDest)
+		defer os.RemoveAll(tmp.Dir)
+		outPath := stdout.AsFile()
+		errPath := stderr.AsFile()
+		os.Link(outPath, filepath.Join(tmp.Dir, "STDOUT"))
```

Comment:
we haven't talked much about the semantic of links (either hard or soft) in IngestDir, so I'd rather, for now, just copy (and yes, that'll be a slight performance hit)